docs/git-node.md

```diff
@@ -316,7 +321,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
```

Comment:
The spaces are here to force a line break in the rendered page.
Before: https://github.com/nodejs/node-core-utils/blob/main/docs/git-node.md#git-node-v8
After: https://github.com/nodejs/node-core-utils/blob/a19b11967dd2f4c562d2824e08e75aa74de3b1ba/docs/git-node.md#git-node-v8

Reply:
good catch! I believe something in my editor auto triggered these removals, updated now!